### ISSUE TITLE
Invalid count argument in resource "ibm_dns_record" promps out when executing the command "openshift-install-powervs create"

### ISSUE THREAD

**Issue by paopao0602** [ocp-power-automation/ocp4-upi-powervs#331]
I am following the guide(https://developer.ibm.com/tutorials/install-ocp-on-power-vs/) to install Red Hat OpenShift Container Platform 4.x on IBM Power Systems Virtual Servers. However the below error occurs. I select IBM Cloud Classic DNS and VPC Load Balancer services. Could you pls kindly have a look?

1. openshift-install-powervs create

```
[init_terraform] Initializing Terraform plugins and validating the code...
[apply] Running terraform apply... please wait
Error: Invalid count argument
  on modules/7_ibmcloud/dns.tf line 56, in resource "ibm_dns_record" "master":
  56:   count              = length(var.master_ips)
The "count" value depends on resource attributes that cannot be determined
until apply, so Terraform cannot predict how many instances will be created.
To work around this, use the -target argument to first apply only the
resources that the count depends on.
Error: Invalid count argument
  on modules/7_ibmcloud/dns.tf line 65, in resource "ibm_dns_record" "worker":
  65:   count              = length(var.worker_ips)

The "count" value depends on resource attributes that cannot be determined
until apply, so Terraform cannot predict how many instances will be created.
To work around this, use the -target argument to first apply only the
resources that the count depends on.
```

My configuration file is as below:
[var.tfvars.log](https://github.com/ocp-power-automation/ocp4-upi-powervs/files/7439125/var.tfvars.log)

**Comment by mkumatag** [comment]
cc @Prajyot-Parab @yussufsh

**Comment by yussufsh** [comment]
@paopao0602 The issue is fixed. For trying the fix on release-4.9 (default ocp version) we will need to wait for https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/333 to be merged.
Once above PR is merged you can delete the `automation` directory from your `openshift-install-powervs` working directory and run the `setup` command again. It will download the latest branch for you.

Just make sure the existing resources are destroyed. I want to warn you that the `automation` directory contains the terraform state file. You may want to backup and restore the `terraform.tfstate` file if "destroy" is failing with the same error.
